data/src/main/java/com/globant/equattrocchio/data/ImagesServicesImpl.java

```diff
@@ -82,4 +108,13 @@ private Image transformImage(ImageResponse imageResponse) {
 
         return images;
     }
+
+    private List<Image> transformCompleteImages(GetLatestImagesResponse getLatestImagesResponse) {
```

Comment:
what is the meaning of "complete" here, I don't get it. If no real meaningful purpose then remove it

Reply:
done